geniza/footnotes/models.py

```diff
@@ -224,6 +224,15 @@ def __str__(self):
         rel = " and ".join([str(choices[c]) for c in self.doc_relation]) or "Footnote"
         return f"{rel} of {self.content_object}"
 
+    def __eq__(self, o):
+        """Footnotes should be considered if content source,
+        location, document relation type, notes, and content all match.
+        Does not include associated content object in comparison check."""
+        return all(
+            getattr(self, val) == getattr(o, val)
+            for val in ["source", "location", "doc_relation", "notes", "content"]
+        )
+
```

Comment:
i think "other" is a little more readable, unless "o" is a django convention or something

Reply:
also i think docstring should say "considered equal"

Reply:
The `o` was from a vscode template that popped in when I started the method! I should have revised, IDK why I didn't. And thanks for catching the missing word equal there